core/src/test/scala/io/finch/response/ResponseBuilderSpec.scala

```diff
@@ -1,71 +1,98 @@
-/*

```

Comment:
I'm using Mac. Only the file's line ending was CRLF, but other files seem to have LF ending.